### ISSUE TITLE
2023 07 20, Accessibility, The background box containing the logo and the heading menu receives focus but does not have an observable function

### ISSUE THREAD

**Issue by jillwillcox** [Block81838/Block81838#1]
Brief Description
The background box containing the logo and the heading menu receives its own focus but does not have an observable function.

Screengrab/videos (insert file names here)
https://www.loom.com/share/bab83dd603b0409aa284dac2a0424044

Link
https://unemployment.oregon.gov/

Environment
(Device > OS > Browser)
Computer > Windows 11 > Google Chrome 114

Type of Bug:  Functional, Accessibility or Content
Accessibility

WCAG Reference Information
2.1.1: Keyboard - https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html

Steps to reproduce
1)	00:00, Launch page in private browser
2)	00:11, using the tab key shift focus through the page
3)	00:19, BUG: focus shifts to the white background behind the logo and navigation menu
a)	This menu cannot be interacted with beyond receiving focus
b)	NOTE: this bug is present on all pages

Desired Result
•	Content receives focus if it is a meaningful image and/or can be interacted with

Actual Result
•	Background element with no purpose or receives focus
